### PR TITLE
Add back button to the coming soon alert popup

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes #161672268

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
This PR is good to go @nerdlet but I see you've added some new stories. Please ensure you've sort them by date descending and **merge**.

**Comment by nerdlet** [comment]
@kilemensi Already merged changes

File: src/components/Showcase/Stories.js [hunk 1]
```diff
@@ -58,13 +59,95 @@ const data = [
       'https://www.theguardian.com/science/2016/jun/09/air-pollution-now-major-contributor-to-stroke?CMP=Share_iOSApp_Other'
   },
   {
+<<<<<<< HEAD
```
Comment (kilemensi) [inline src/components/Showcase/Stories.js]: There is a bad/pending merge here @nerdlet.

**Comment by nerdlet** [comment]
@kilemensi My bad, had not noticed that. Made the necessary change

**Pull request merged**